packages/terra-clinical-item-view/tests/jest/__snapshots__/ItemView.test.jsx.snap

```diff
@@ -2,8 +2,6 @@ exports[`test should render 1 start theme display 1`] = `
 <div
   className="terraClinical-ItemView terraClinical-ItemView--oneColumn terraClinical-ItemView-accessory--alignCenter">
```

Comment:
`terraClinical-ItemView-accessory--alignCenter` is still assigned to the className even when no accessory is present. Should this happen?